src/marketing/customer-segment-create.md

```diff
@@ -11,6 +11,13 @@ _Customer Segments list_
 
 The following example shows how to create a customer segment that targets female customers in Los Angeles.
 
+|Column|Description|
+|--- |--- |
+|ID|A unique, sequential number that is assigned when a new segment is saved for the first time.|
+|Segment|The name of the segment.|
+|Status|The current segment status. Options include: Active / Inactive.|
+|Website|The website where the segment applies.|
+
```

Comment:
This is disruptive in this location and not really needed. Let's just keep the lists to the subsections below.

```suggestion
```

Reply:
Thank you for your advice! Done ✅ 
@leslietilling